entity-management-web/src/main/java/eu/europeana/entitymanagement/web/service/ConceptSchemeService.java

```diff
@@ -55,18 +55,22 @@ public ConceptScheme createConceptScheme(ConceptScheme scheme) throws SolrServic
     
     setMandatoryFields(scheme);
 
-    ConceptScheme dbScheme = emConceptSchemeRepo.saveConceptScheme(scheme);
-    setConceptSchemeId(dbScheme);
+    setConceptSchemeId(scheme);
     
+    return storeConceptScheme(scheme);
+  }
+
+  public ConceptScheme storeConceptScheme(ConceptScheme scheme) throws SolrServiceException {
+    emConceptSchemeRepo.saveConceptScheme(scheme);
     try {
       solrService.storeConceptScheme(new SolrConceptScheme(scheme));
     } catch (SolrServiceException e) {
       throw new SolrServiceException(
```

Comment:
the indexing exception should be logged with warning level only. There should be no error response if the concept scheme was successfully written into the DB